site/content/en/docs/start/_index.md

```diff
@@ -455,7 +455,8 @@ choco install minikube
       [Environment]::SetEnvironmentVariable('Path', $('{0};C:\minikube' -f $oldPath), [EnvironmentVariableTarget]::Machine) `
     }
     ```
-    _If you used a CLI to perform the installation, you will need to close that CLI and open a new one before proceeding._
+    <span style="color:blue">
+    _If you used a CLI to perform the installation, you will need to close that CLI and open a new one before proceeding._</span>
```

Comment:
Thank you for this PR, 
it seems like after changing this PR the "_" no longer makes the font "italic"

we could do replace "_" with "*" 

"If you used a CLI to perform the installation, you will need to close that CLI and open a new one before proceeding."

Reply:
also how about we change the wording to
"If you used a terminal (like powershell) for the installation, you will need to close the terminal and reopen it before running minikube."